installers/linux-shared/before-remove.sh.erb

```diff
@@ -20,6 +20,6 @@
     /etc/init.d/<%= name %> stop 2>&1
     /sbin/chkconfig --del <%= name %>
 <% elsif type == 'deb' -%>
-    invoke-rc.d <%= name %> stop
+    invoke-rc.d <%= name %> stop || true
```

Comment:
@arvindsv — Should we be doing a `update-rc.d remove` — the equivalent of `chkconfig --del`?

I remember not adding it here, because it was handled in the `after-remove` script for debian packages.